Sources/SwiftSyntaxBuilder/generated/ResultBuildersFile.swift

```diff
@@ -16,73 +17,61 @@ import SwiftSyntax
 
 @resultBuilder
 public struct CodeBlockItemListBuilder {
-
```

Comment:
Is there a way with the new API to add new lines, without should adding a empty line at the beginning of each `Decl`